plone/app/relationfield/__init__.py

```diff
@@ -1,14 +1,13 @@
+from importlib.metadata import distribution
+from importlib.metadata import PackageNotFoundError
 from plone.app.relationfield.monkey import PATCHES
 
-import pkg_resources
-
 
 PATCHES
 
 
 try:
-    pkg_resources.get_distribution("plone.app.contenttypes")
-except pkg_resources.DistributionNotFound:
-    HAS_CONTENTTYPES = False
-else:
+    distribution("plone.app.contenttypes")
     HAS_CONTENTTYPES = True
+except PackageNotFoundError:
+    HAS_CONTENTTYPES = False
```

Comment:
We should be able to remove this; there is no Plone without plone.app.contenttypes now.

Reply:
good idea 👍🏾